cmd/vela-terraform/plan.go

```diff
@@ -131,23 +131,20 @@ func (p *Plan) Command(dir string) *exec.Cmd {
 
 	// check if Vars is provided
 	if len(p.Vars) > 0 {
-		var vars string
 		for _, v := range p.Vars {
-			vars += fmt.Sprintf("-var=%s ", v)
+			flag := fmt.Sprintf(`-var=%s`, v)
+			// add flag for Vars from provided command
+			flags = append(flags, flag)
```

Comment:
Do you think we could condense this into one line?

```suggestion
			// add flag for Vars from provided command
			flags = append(flags, fmt.Sprintf(`-var=%s`, v))
```